lib/evmone/instructions.cpp

```diff
@@ -103,7 +106,7 @@ void op_div(execution_state& state, instr_argument) noexcept
 void op_sdiv(execution_state& state, instr_argument) noexcept
 {
     auto& v = state.stack[1];
-    v = v != 0 ? intx::sdivrem(state.stack[0], v).quot : 0;
+    v = v != 0 ? sdivrem(state.stack[0], v).quot : 0;
```

Comment:
I actually like the explicit namespace because it improves readability in many places.

Reply:
Me too, especially for things like `be::store(..., ...)` it would be clearer, that some conversion is going on

Reply:
I brought back the `be::...` but `intx::be::load<intx::uint256>` seems a bit too much...

Reply:
The `intx::sdivrem()` is never needed, because of ADL. I can bring it back in similar cases, but I cannot promise it will stay consistent over time.